snmp/tests/test_profiles.py

```diff
@@ -246,13 +248,49 @@ def test_3850(aggregator):
         for metric in CIE_METRICS:
             aggregator.assert_metric('snmp.{}'.format(metric), metric_type=aggregator.GAUGE, tags=tags, count=1)
         aggregator.assert_metric('snmp.cieIfResetCount', metric_type=aggregator.MONOTONIC_COUNT, tags=tags, count=1)
+
     for temp in range(3):
         for switch in range(1, 3):
             env_tag = ['temp_descr:Switch {} - Temp Sensor {}, GREEN '.format(switch, temp)]
             aggregator.assert_metric(
                 'snmp.ciscoEnvMonTemperatureStatusValue', metric_type=aggregator.GAUGE, tags=env_tag + common_tags
             )
-    aggregator.assert_metric('snmp.sysUpTimeInstance', count=1)
+
+    for switch in range(1, 3):
+        for supply, status in [('A', 'Normal'), ('B', 'NotExist')]:
+            env_tags = ['power_supply_descr:Switch {} - Power Supply {}, {}'.format(switch, supply, status)]
```

Comment:
Are these normalized?

Reply:
I don't believe those tags are normalized. After speaking with @naaltman, I've removed those tags and replaced with alternatives where applicable.